connector/client/client.go

```diff
@@ -1,5 +1,122 @@
 package client
 
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+	"fmt"
+	"errors"
+
+	"github.com/infracloudio/msbotbuilder-go/connector/auth"
+	"github.com/infracloudio/msbotbuilder-go/schema"
+	"github.com/infracloudio/msbotbuilder-go/schema/customerror"
+)
+
 type Client interface {
 	Post(url url.URL, activity schema.Activity) error
 }
+
+type ConnectorClient struct {
+	Config ConnectorClientConfig
```

Comment:
We don't need to declare new field here. We can just embed Config as: https://github.com/infracloudio/msbotbuilder-go/blob/develop/core/bot_framework_adapter.go#L34